examples/galaxies/plot_photometry.py

```diff
@@ -0,0 +1,90 @@
+"""
+Optical Photometry
+==================
+
+This example demonstrates how to model galaxy photometric magnitudes using the
+kcorrect spectral energy distribution templates as implemented in SkyPy.
+
+"""
+
+# %%
+# kcorrect Spectral Templates
+# ---------------------------
+#
+# In SkyPy, the rest-frame spectral energy distributions (SEDs) of galaxies can
+# be modelled as a linear combination of the five kcorrect basis templates
+# [1]_. One possible model for the coefficients is a redshift-dependent
+# Dirichlet distribution [2]_ which can be sampled from using the
+# :func:`dirichlet_coefficients <skypy.galaxy.spectrum.dirichlet_coefficients>`
+# function. The coefficients are then taken by the :meth:`kcorrect.absolute_magnitudes
+# <skypy.galaxy.spectrum.SpectrumTemplates.absolute_magnitudes>` and
+# :meth:`kcorrect.apparent_magnitudes
+# <skypy.galaxy.spectrum.SpectrumTemplates.apparent_magnitudes>`
+# methods to calculate the relevant photometric quantities using the
+# :doc:`speclite <speclite:overview>` package. Note that since the kcorrect
+# templates are defined per unit stellar mass, the total stellar mass of each
+# galaxy must either be given or calculated from its absolute magnitude in
+# another band using :meth:`kcorrect.stellar_mass
+# <skypy.galaxy.spectrum.KCorrectTemplates.stellar_mass>`.
+# An example simulation for the SDSS u- and r-band apparent magnitudes of "red"
+# and "blue" galaxy populations is given by the following config file:
+#
+# .. literalinclude:: ../../../examples/galaxies/sdss_photometry.yml
+#   :language: YAML
+#   :caption: examples/galaxies/sdss_photometry.yml
+#
+# The config file can be downloaded
+# `here <https://github.com/skypyproject/skypy/raw/master/examples/galaxies/sdss_photometry.yml>`_
```

Comment:
since you are already explicitly going up three levels for the `literalinclude`, you could try to see if this works:
```suggestion
# The config file can be downloaded
# :download:`here <../../../examples/galaxies/sdss_photometry.yml>`
```